### PR TITLE
Toponaming/Part:  move in  AppPartPy

### PR DESCRIPTION
**Pull request by bgbsww**

### CODE REVIEW

File: src/Mod/Part/App/AppPartPy.cpp [hunk 1]
```diff
@@ -106,6 +106,8 @@
 
 #ifdef FCUseFreeType
 #  include "FT2FC.h"
+#include "TopoShapeOpCode.h"
+#include "TopoShapeMapper.h"
```
Comment (chennes) [inline src/Mod/Part/App/AppPartPy.cpp:110]: Is it really correct that these includes are inside this ifdef? They seem unrelated to FreeType.

File: src/Mod/Part/App/AppPartPy.cpp [hunk 2]
```diff
@@ -885,8 +887,34 @@ class Module : public Py::ExtensionModule<Module>
         }
         return list;
     }
-    Py::Object makeCompound(const Py::Tuple& args)
+    Py::Object makeCompound(const Py::Tuple& args, const Py::Dict &kwds)
```
Comment (chennes) [inline src/Mod/Part/App/AppPartPy.cpp:890]: Why switch to the keyword-argument version if kwds is not used?

File: src/Mod/Part/App/AppPartPy.cpp (hunk 1 above)
Comment (bgbsww) [reply src/Mod/Part/App/AppPartPy.cpp:110]: CLion, I am never again trusting your cute resolve missing definitions button.  I literally never looked up there.  My bad.
